Sources/RichTextKit/RichTextView_AppKit.swift

```diff
@@ -77,8 +95,10 @@ open class RichTextView: NSTextView, RichTextViewComponent {
      */
     open func setup(
         with text: NSAttributedString,
-        format: RichTextDataFormat
+        format: RichTextDataFormat,
+        scrollingDisabled: Bool = false
```

Comment:
hmm  this is good point. I will just add it to my configuration later https://github.com/DominikBucher12/RichTextKit/pull/1  Thanks for this!

Reply:
I have implemented inline support for the RTE (macOS) including an inline demo. Please look at my fork:
https://github.com/screenworker/RichTextKit/tree/rte-inline-support